zerver/webhooks/newrelic/tests.py

```diff
@@ -4,40 +4,123 @@
 class NewRelicHookTests(WebhookTestCase):
     STREAM_NAME = 'newrelic'
     URL_TEMPLATE = "/api/v1/external/newrelic?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'newrelic'
 
-    def test_alert(self) -> None:
-        expected_topic = "Apdex score fell below critical level of 0.90"
-        expected_message = 'Alert opened on [application name]: Apdex score fell below critical level of 0.90 ([view alert](https://rpm.newrelc.com/accounts/[account_id]/applications/[application_id]/incidents/[incident_id])).'
+    def test_open(self) -> None:
+        expected_topic = "New Relic Alert - Test Policy (1234)"
```

Comment:
Just to be clear, the "New Relic Alert" - Test Policy" is a name you entered in configuring New Relic, and doesn't have to contain "New Relic", right?  Generally or policy is to not repeat the name of a product here because it's usually implied by the name of the bot sending the message.

Reply:
Correct, I will change the tests to reflect this policy and be more clear.